lib/twine-ui/src/components/Typography.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from 'styled-components';
```

Comment:
This was added a while ago, is this still consistent with (1) any updates to the design system and (2) the equivalent components in the dashboard?